libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -87,7 +86,7 @@ class BitcoinLikeExplorer implements IExplorer {
     client.interceptors.response.use(responseInterceptor, errorInterceptor);
   }
 
-  async broadcast(tx: string): Promise<any> {
+  async broadcast(tx: string): Promise<AxiosResponse<any, any>> {
```

Comment:
I find it weird to return an AxiosResponse ? is it not possible to just return the type of data you want, or we are not sure what we are going to get at this point ?

Reply:
Ideally the expected response should be typed, if possible.

Reply:
fixed